internal/saucecloud/espresso.go

```diff
@@ -257,9 +257,14 @@ func (r *EspressoRunner) newStartOptions(
 		// Overwrite device settings
 		RealDeviceKind: strings.ToLower(espresso.Android),
 		AppSettings: job.AppSettings{
-			AudioCapture: s.AppSettings.AudioCapture,
+			InstrumentationEnabled: s.AppSettings.ResigningEnabled,
```

Comment:
Isn't that supposed to be `InstrumentationEnabled`?
```suggestion
			InstrumentationEnabled: s.AppSettings.InstrumentationEnabled,
```

Reply:
This `AppSettings` to `job.AppSettings` struct mapping maps saucectl yaml values to API call payload json. In the yaml, the definition `ResigningEnabled` is used both for iOS and Android but in the API call, the name is different per platform. I wish I was able to rename it so that it's consistent but that's a breaking change for existing saucectl users, so we do this mapping as the lesser evil.

TLDR: The shape of that line is intentional, to avoid breaking all YAML files out there.

Reply:
We're updating the docs today 👍

Reply:
Thanks @diegoperini! It'd be good to leave an inline comment in the code explaining that little difference. I won't remember this in 2 weeks (or any new dev that joins) 😂

Reply:
Good idea, added it.